oshmem/mca/sshmem/sshmem_types.h

```diff
@@ -112,9 +113,10 @@ typedef struct map_segment {
     sshmem_mkey_t       *mkeys;          /* includes local segment bases in va_base */
     segment_flag_t       flags;          /* enable/disable flag */
     int                  seg_id;
-    char                 seg_name[OPAL_PATH_MAX];
```

Comment:
@alex-mikheev do you believe that `seg_name` is needless? I see that original code has `unlink(ds_buf->seg_name)` in `shmem_ds_reset()`. I do not remember if it is something useful but pay attention to avoid related issues.